src/components/vedtak/VedtakInfopanel.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Innholdstittel } from 'nav-frontend-typografi';
+import { restdatoTildato } from '../../utils/datoUtils';
+import { VedtakDTO } from '../../reducers/vedtak';
+import Panel from 'nav-frontend-paneler';
+import VedtakOppsummering from './VedtakOppsummering';
+import VedtakUtbetaltePerioder from './VedtakUtbetaltePerioder';
+import styled from 'styled-components';
+import { Row } from 'nav-frontend-grid';
+
+
+interface IVedtakInfopanel {
+    selectedVedtak: VedtakDTO,
+}
+
+const StyledPanel = styled(Panel)`
+    padding: 1.5em
+`;
+
+const VedtakInfopanel = (vedtakIProps: IVedtakInfopanel) => {
+    const { selectedVedtak } = vedtakIProps;
+
+    return (
+        <StyledPanel>
+            <Row>
+                <Innholdstittel>
+                    {`${restdatoTildato(selectedVedtak.vedtak.fom)} - ${restdatoTildato(selectedVedtak.vedtak.tom)}`}
```

Comment:
Det gjør ingenting, men jeg tror det finnes en "datoer til lesbar periode"-metode, hvis du ikke vil lage periode selv.

Reply:
Det var noen problemer med datoen som kommer inn fra spinnsyn. Ser ikke ut som om den klarer å gjøre det om til date. Denne gjør at datoen blir med prikker istedenfor bindestrek

Reply:
Den er grei! De util-metodene for datoer har blitt litt hjemmesnekra etter hvert som man har fått behov for det! 🛠️

Reply:
Jeg tenkte først restdatoTildato var noe annet, men så innså jeg at jeg ikke helt vet rest henviser til - er det overskuddsdager eller REST, og er det avstand mellom to datoer, eller konvertering mellom datoformat?

Reply:
Gode spørsmål! Dette er REST som i ikke-SOAP, altså at den forventer noe ala "2020-11-19:12:00T+01:00" og skal gi deg en streng med "dd MM ÅÅÅ"